classes/ValidFormBuilder/MultiField.php

```diff
@@ -147,6 +148,8 @@ public function __construct($label, $meta = array())
 
         $this->__dynamic = $this->getMeta("dynamic", $this->__dynamic);
         $this->__dynamicLabel = $this->getMeta("dynamicLabel", $this->__dynamicLabel);
+
+        $this->initialiseDynamicRemoveMeta();
```

Comment:
I'll look into that. Regarding this:

> There are no dynamic fields that cannot be removed.

At the moment, *if* you add the 'dynamicRemoveLabel' logic, the remove logic kicks in. If you don't add that label, the original dynamic field logic still stands. (Not-requiring duplicated required fields etc.) I figured that would allow for best backwards compatibility.